test/CMakeLists.txt

```diff
@@ -751,6 +753,15 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_viirs_aod
                           -n 0.0"
                           viirs_aod.nc ${IODA_CONV_COMP_TOL_ZERO})
 
+ecbuild_add_test( TARGET  test_${PROJECT_NAME}_satbias_viirs_aod
+                    TYPE    SCRIPT
```

Comment:
Pedantic but can you make the indentation match the above and below?

Reply:
sure.